### ISSUE TITLE
Tracking issue for pinning to a hyper version

### ISSUE THREAD

**Issue by moosingin3space** [moosingin3space/hyproxy#5]
When Hyper makes a release that includes tokio support, pin its version in `Cargo.toml`.

**Comment by dekellum** [comment]
Hyper 0.12 has been released. Sufficient?

**Comment by moosingin3space** [comment]
Yes, it would be. There's a fair amount of work that has to be done to get this up-to-date, and I'll probably wait until async/await is in stable to do a major refactor.

**Comment by dekellum** [comment]
Yep, that will be a major refactor for body-image (currently on tokio 0.1.x and futures 0.1.x ) as well.
